modules/alloy/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Added
```

Comment:
This doesn't look like an `alloy` change? I suspect this was meant for the TinyMCE changelog as we're missing a changelog there. We'd then still need one here for the `onToggled` callback added.

- [ ] Make sure an appropriate changelog exists for `alloy` and `tinymce`